media-libs/webvfx/webvfx-1.1.0.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit flag-o-matic qmake-utils xdg
+
+DESCRIPTION="Video effects library based on web technologies"
+HOMEPAGE="https://github.com/mltframework/webvfx/"
+SRC_URI="https://github.com/mltframework/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+IUSE=""
+
+KEYWORDS="~amd64 ~x86"
```

Comment:
Per `skel.ebuild` template, this is between SLOT and IUSE as one block without empty lines, we should follow that example.